### ISSUE TITLE
Add examples and proper readme 

### ISSUE THREAD

**Issue by kunal-dd** [delta-golang/delta-go#1]
Hey,

Can you please add some examples and proper guide to use this package?

**Comment by arashbina** [comment]
@KunalDholiya absolutely. This is very much in early stages and a work in progress. Adding examples, tests and documentation over the next few weeks.
